pkgs/leak_tracker/lib/src/usage_tracking/model.dart

```diff
@@ -174,15 +167,4 @@ class AutoSnapshottingConfig {
 
   /// The absolute path to the [directory].
   String get directoryAbsolute => path.absolute(directory);
-
-  @override
-  String toString() {
-    final formatter = NumberFormat('#,###,000');
```

Comment:
As a final comment, I have no concerns whatsoever if we keep these toString() methods, we'd just likely want a slightly less sophisticated way of formatting numbers.